CMakeLists.txt

```diff
@@ -373,39 +373,6 @@ IF (INSTALL_LAYOUT)
     ADD_DEPENDENCIES(funcexp GenError)
     ADD_DEPENDENCIES(oamcpp GenError)
     ADD_DEPENDENCIES(regr_mysql GenError)
-    # Don't know why, but this doesn't work if you make the above line PARENT_SCOPE
-    SET(CPACK_COMPONENTS_ALL ${CPACK_COMPONENTS_ALL} PARENT_SCOPE)
-    IF (RPM)
-        SET(ignored
-        "%ignore ${CMAKE_INSTALL_PREFIX}"
-        "%ignore ${CMAKE_INSTALL_PREFIX}/local"
-        "%ignore ${CMAKE_INSTALL_PREFIX}/bin"
-        "%ignore ${CMAKE_INSTALL_PREFIX}/lib"
-        "%ignore ${CMAKE_INSTALL_PREFIX}/sbin"
-        "%ignore ${CMAKE_INSTALL_PREFIX}/lib64/mysql"
-        "%ignore ${CMAKE_INSTALL_PREFIX}/lib64/mysql/plugin"
-        "%ignore /var/lib"
-        "%ignore /var"
-        )
-        SET(CPACK_RPM_columnstore-engine_USER_FILELIST ${ignored} "%config(noreplace) ${INSTALL_SYSCONF2DIR}/*" PARENT_SCOPE)
-
-        SET(SUSE_VERSION_NUMBER OFF)
-        IF (EXISTS "/etc/SuSE-release")
-            file (READ "/etc/SuSE-release" SUSE_VERSION)
-            string(REGEX MATCH "VERSION = ([0-9]+)" SUSE "${SUSE_VERSION}")
-            set(SUSE_VERSION_NUMBER "${CMAKE_MATCH_1}")
-        ENDIF ()
-        if (${SUSE_VERSION_NUMBER} EQUAL 12)
-            SETA(CPACK_RPM_columnstore-engine_PACKAGE_REQUIRES "expect" "boost-devel >= 1.54.0" "snappy" "jemalloc" "net-tools MariaDB-server" PARENT_SCOPE)
-        else ()
-            SETA(CPACK_RPM_columnstore-engine_PACKAGE_REQUIRES "expect" "boost >= 1.53.0" "snappy" "jemalloc" "net-tools MariaDB-server" PARENT_SCOPE)
-        endif()
-
-        SET(CPACK_RPM_columnstore-engine_PRE_INSTALL_SCRIPT_FILE ${CMAKE_CURRENT_SOURCE_DIR}/build/preInstall_storage_engine.sh PARENT_SCOPE)
-        SET(CPACK_RPM_columnstore-engine_POST_INSTALL_SCRIPT_FILE ${CMAKE_CURRENT_SOURCE_DIR}/build/postInstall_storage_engine.sh PARENT_SCOPE)
-        SET(CPACK_RPM_columnstore-engine_PRE_UNINSTALL_SCRIPT_FILE ${CMAKE_CURRENT_SOURCE_DIR}/build/preUn_storage_engine.sh PARENT_SCOPE)
-
-    ENDIF ()
```

Comment:
removing this probably breaks our standalone builds.  We do that often to save time.  If it's not hurting anything I'd prefer to leave it in.